cli/tests/integration/run_tests.rs

```diff
@@ -826,15 +826,15 @@ itest!(config {
 
 itest!(config_types {
   args:
-    "run --reload --quiet --config run/config_types/tsconfig.json run/config_types/main.ts",
+    "run --reload --quiet --config --check=all run/config_types/tsconfig.json run/config_types/main.ts",
```

Comment:
This regression was not caught because these tests weren't type checking due to the change to not type check by default.